sklearn/metrics/regression.py

```diff
@@ -241,7 +241,17 @@ def mean_squared_error(y_true, y_pred,
     return np.average(output_errors, weights=multioutput)
 
 
-def median_absolute_error(y_true, y_pred):
+def _weighted_percentile(array, sample_weight, percentile=50):
```

Comment:
I think we can rename this to weighted_median, remove the percentile argument and move it to utils.extmath